pkg/util/shell/cat.go

```diff
@@ -0,0 +1,44 @@
+package shell
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+)
+
+func Cat(arguments []string, stdin io.Reader, stdout io.Writer) error {
+
+	if len(arguments) == 1 {
+		_, err := io.Copy(stdout, stdin)
+		if err != nil {
+			return fmt.Errorf("cat: %v", err)
+		}
+		return nil
+	}
+
+	for i := 1; i < len(arguments); i++ {
+		filename := arguments[i]
+		err := printFile(filename, stdin, stdout)
+		if err != nil {
+			return fmt.Errorf("cat: %v", err)
```

Comment:
does this return the correct cat error? (e.g. the same as `cat not-found.txt`)

Reply:
yes